### PR TITLE
Add alpha taxonomy 'exists' endpoint

### PR DESCRIPTION
**Pull request by gwarmstrong**
This PR adds an 'exists' functionality for alpha diversity and taxonomy resource, which allow you to query individual or groups of samples for whether they are contained the corresponding resource. As mentioned in (and blocking) https://github.com/biocore/microsetta-interface/pull/11

### CODE REVIEW

File: microsetta_public_api/api/microsetta_public_api.yml [hunk 1]
```diff
@@ -401,6 +442,47 @@ paths:
...
+      tags:
+        - Taxonomy
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/sampleIdArray"
+      responses:
+        '200':
+          description: Indicates whether each sample is in the taxonomy resource, in the same order as the input list
```
Comment (wasade) [inline microsetta_public_api/api/microsetta_public_api.yml:476]: `False` means it either does not exist or the order is different?

File: microsetta_public_api/api/taxonomy.py [hunk 2]
```diff
@@ -76,3 +76,24 @@ def _present_microbes_taxonomy_table(sample_ids, resource):
     taxonomy_table = taxonomy_.presence_data_table(sample_ids)
     response = jsonify(taxonomy_table.to_dict())
     return response, 200
+
+
+def exists_single(resource, sample_id):
```
Comment (wasade) [inline microsetta_public_api/api/taxonomy.py:81]: this and the code below look the same as what was done with alpha, should it be decomposed?

File: microsetta_public_api/api/microsetta_public_api.yml (hunk 1 above)
Comment (gwarmstrong) [reply microsetta_public_api/api/microsetta_public_api.yml:476]: `False` means it does not exist in the taxonomy resource. 
e.g., `/taxonomy/exists/table #=> ['sample-1', 'sample-2', 'sample-3']`
If that returned `[true, false, true]`, then it means `sample-2` is not in `table`. The order within the resource itself is irrelevant.

Can change the wording to make this more clear.

Comment (gwarmstrong) [inline microsetta_public_api/api/microsetta_public_api.yml:476]:
```suggestion
          description: Indicates whether each sample is in the taxonomy resource. Order of the output array corresponds to the order of the input array.
```

File: microsetta_public_api/api/taxonomy.py (hunk 2 above)
Comment (gwarmstrong) [reply microsetta_public_api/api/taxonomy.py:81]: I think its fine for now... Its only 7 lines of repeated code, and the decomposition, which we did elsewhere within `validate_resource`, I believe, is much less readable and necessarily couples the interface for `TaxonomyRepo` and `AlphaRepo` which could either be nice, or really annoying later...

Comment (wasade) [reply microsetta_public_api/api/taxonomy.py:81]: okay

**Pull request merged**